### ISSUE TITLE
Question about the training detail on Qwen2-VL

### ISSUE THREAD

**Issue by xin-li-67** [OSU-NLP-Group/UGround#25]
Hi UGround Team,

Thanks for the great work and the detailed technical paper. However, I have a few questions about your training process on Qwen2-VL.

In Appendix F.3 of your technical report, you mentioned that the pertaining process is identical to the original LLaVA-1.5. You changed the grounding data from normalized coordinate values to absolute values for the fine-tune stage with your modified AnyRes method. Are you following the same fine-tune recipe for the Qwen2-VL-Instruct models on these grounding data? Is there any performance fluctuation?

I am doing something similar by changing the coordinates format from the one we used for our pre-trained model to a new in-house format. However, this will result in a huge performance degradation compared with our PT checkpoints on grounding tasks like the RefCOCO-test.

I have tried your hyperparameters, but the issue still occurs.

Best,

**Comment by boyugou** [comment]
We also provided some details in https://github.com/OSU-NLP-Group/UGround/tree/main/train/qwen2_vl
